src/System.Runtime/tests/TimeZoneTests.netstandard1.7.cs

```diff
@@ -0,0 +1,132 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class TimeZoneTests
+    {
+        [Fact]
+        public static void TestBasicTimeZoneProperties()
+        {
+            var currentZone = TimeZone.CurrentTimeZone;
+            Assert.Equal(TimeZoneInfo.Local.StandardName, currentZone.StandardName);
+            Assert.Equal(TimeZoneInfo.Local.DaylightName, currentZone.DaylightName);
+
+            int year = DateTime.Today.Year;
+
+            DaylightTime dt = currentZone.GetDaylightChanges(year);
+
+            TimeZoneInfo.AdjustmentRule currentRule = null;
+            if (TimeZoneInfo.Local.SupportsDaylightSavingTime)
+            {
+                foreach (var rule in TimeZoneInfo.Local.GetAdjustmentRules())
+                {
+                    if (rule.DateStart.Year <= year && rule.DateEnd.Year >= year && rule.DaylightDelta != TimeSpan.Zero)
+                    {
+                        currentRule = rule;
+                        break;
+                    }
+                }
+            }
+
+            if (currentRule != null)
```

Comment:
Yes TimeZone class works with the Local TZ only so there is no way we can test it with something different than local TZ. but in general I am not expecting a problem in the test even when running under different time zones settings.